patterns/core/ProductDetailInformation/Buybox.js

```diff
@@ -45,11 +45,29 @@ export default function Buybox(props) {
   function handleAddToCart() {
     setLoading(true)
 
+    GTM.trackEvent({
+      event: 'add_to_cart',
+      ecommerce: {
+        items: variants.map(prepareTrackingItem),
+      },
+      _clear: true,
+    })
+
     setTimeout(() => {
       setLoading(false)
     }, 3000)
   }
 
+  function handleAddToWishlist() {
+    GTM.trackEvent({
+      event: 'add_to_wishlist',
+      ecommerce: {
+        items: variants.map(prepareTrackingItem),
```

Comment:
Same as for the addToCart event, we are only adding one product/variant to the wishlist, not multiple.